docs/changelog.md

```diff
@@ -23,6 +23,9 @@ permalink: /changelog/
 
 * **feature-syncedtabs**
   * ⚠️ **This is a breaking change**: Adds context to the constructor of `SyncedTabsFeature`.
+  * ⚠️ **This is a breaking change**: Removed the following properties from `DownloadJobState` in `AbstractFetchDownloadService` and added to `DownloadState`: `DownloadJobStatus` (now renamed to `DownloadStatus`) and `currentBytesCopied`. These properties can now be read from `DownloadState`.
+    * ⚠️ **This is a breaking change**: Removed the enum class `DownloadJobStatus` from `AbstractFetchDownloadService` and moved into `DownloadState`, and removed the `ACTIVE` state while introducing two new states called `INITIATED` and `DOWNLOADING`.
```

Comment:
The description looks  good 👍  lets add another section for * **feature-downloads** as this section belongs to **feature-syncedtabs**

Reply:
Ok, is it good now?

Reply:
Looking good, let's just remove the extra indentation on line 27 `    * ⚠️ **This is a breaking change**:` :)
<img width="1169" alt="image" src="https://user-images.githubusercontent.com/773158/88711133-b28e9b80-d0e5-11ea-80c0-b27b493aac52.png">